client/src/components/App/Content/LandingPage/adminLandingPage/UnallocatedCard/UnallocatedCardStyles.ts

```diff
@@ -10,6 +12,10 @@ const useStyles = makeStyles({
         borderRadius: '1vw',
         padding: '1vw',
         cursor: 'pointer',
+        color: theme.palette.text.secondary,
+        '&:hover': {
+            color: theme.palette.text.primary
+        }
```

Comment:
maybe have generic styles hook for all cards and put this in one of the classes?